docs/sources/flow/reference/components/otelcol.processor.attributes.md

```diff
@@ -164,6 +164,10 @@ For example, adding a `span_names` filter could cause the component to error if
 
 The `exclude` block provides an option to exclude data from being fed into the [action] blocks based on the properties of a span, log, or metric records.
 
+> **NOTE**: Signals excluded by the `exclude` block will still be propagated to downstream components.
+> If you would like to not propagate certain signals to downstream components, 
+> consider a processor such as [otelcol.processor.tail_sampling](../otelcol.processor.tail_sampling/).
```

Comment:
```suggestion
{{% admonition type="note" %}}
Signals excluded by the `exclude` block are still be sent to downstream components.
To prevent specific signals from being sent to downstream components,  you can use a processor
such as [otelcol.processor.tail_sampling](../otelcol.processor.tail_sampling/).
{{% /admonition %}}
```

Better to use the admonition shortcode for notes where we can.